### PR TITLE
Add registration attention messages

### PR DESCRIPTION
**Pull request by pdeziel**
### Scope of changes

This adds a few new messages to the attention endpoint to facilitate better communication to the user:

`VerifyEmails`: When the registration for a network has been submitted but contact emails need to be verified
`RegistrationPending`: When the registration is awaiting review from the TRISA admins and certificate issuance
`RegistrationApproved`: When the registration has been approved and the certificate has been issued

This also cleans up some of the message text to be consistent with the frontend text.

SC-7902

### Type of change

- [ ] bug fix
- [x] new feature
- [ ] documentation
- [ ] other (describe)

### Acceptance criteria

The new states make sense and are useful to the user.

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [x]   Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.

### CODE REVIEW

File: pkg/bff/admin.go [hunk 1]
```diff
@@ -207,22 +210,29 @@ func (s *Server) GetVASPs(ctx context.Context, testnetID, mainnetID string) (tes
...
 }
 
-// certificateMessage returns a certificate attention message for the given VASP, if
-// applicable. The states are distinct, so at most one message is returned.
-func certificateMessage(vasp *pb.VASP, network string) (msg *api.AttentionMessage, err error) {
-	const expireLayout = "February 2, 2006"
+// registrationMessage returns a message corresponding to the registration state of the
+// VASP. These states are distinct, so only one message is returned.
+func registrationMessage(vasp *pb.VASP, network string) (msg *api.AttentionMessage, err error) {
+	const expireLayout = "January 2, 2006"
```
Comment (pdeziel) [inline pkg/bff/admin.go:216]: Correcting the custom layout so the date is formatted correctly.

File: pkg/bff/admin.go [hunk 2]
```diff
@@ -136,11 +136,14 @@ func (s *Server) Certificates(c *gin.Context) {
...
 	RenewCertificate     = "Your organization's %s X.509 Identity Certificate will expire on %s. Start the renewal process to receive a new X.509 Identity Certificate and remain a trusted member of the TRISA network."
-	CertificateRejected  = "Your organization's %s X.509 Identity Certificate has been rejected by TRISA. This means your organization is not a verified member of the TRISA network and cannot communicate with other members. Please contact TRISA at support@trisa.io for additional details and next steps."
-	CertificateRevoked   = "Your organization's %s X.509 Identity Certificate has been revoked by TRISA. This means your organization is no longer a verified member of the TRISA network and can no longer communicate with other members. Please contact TRISA at support@trisa.io for additional details and next steps."
+	CertificateRevoked   = "Your organization's %s X.509 Identity Certificate has been revoked by TRISA. This means your organization is no longer a verified member of the TRISA network and can no longer communicate with other members. Please contact TRISA support at support@trisa.io for additional details and next steps."
```
Comment (rebeccabilbro) [inline pkg/bff/admin.go:146]: I recommend we change "support@trisa.io" to "support@rotational.io" (see also [this story](https://app.shortcut.com/rotational/story/7789/updated-contact-email-in-templates)) -- I don't think the admin@trisa.io email address actually works anymore, and I've never heard of the support@trisa.io one. CC: @eschmier

File: proto/bff/models/v1/models.proto [hunk 3]
```diff
@@ -138,10 +138,12 @@ enum AttentionSeverity {
 // AttentionAction is used to indicate the action that should be taken by the user in
 // response to an attention message.
 enum AttentionAction {
-    START_REGISTRATION = 0;
-    COMPLETE_REGISTRATION = 1;
-    SUBMIT_TESTNET = 2;
-    SUBMIT_MAINNET = 3;
-    RENEW_CERTIFICATE = 4;
-    CONTACT_SUPPORT = 5;
+    NO_ACTION = 0;
```
Comment (rebeccabilbro) [inline proto/bff/models/v1/models.proto:141]: Guessing we're intentionally using the 0-val of the enum to do nothing by default if no val is passed in?

**Review by rebeccabilbro — APPROVED** [review]
One small change related to the support email provided, but otherwise looks good.

File: proto/bff/models/v1/models.proto (hunk 3 above)
Comment (pdeziel) [reply proto/bff/models/v1/models.proto:141]: It's not really a default value, but we do need a value to represent the "no action required" messages, such as when a registration is approved.

File: pkg/bff/admin.go (hunk 2 above)
Comment (pdeziel) [reply pkg/bff/admin.go:146]: Ah, good call!

**Pull request merged**